src/Microsoft.DotNet.Interactive.PowerShell/Commands/OutDisplayCommand.cs

```diff
@@ -1,6 +1,7 @@
 // Copyright (c) .NET Foundation and contributors. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 using static Microsoft.DotNet.Interactive.Kernel;
+using System.Collections.Generic;
 
```

Comment:
nit: can you also move the following 2 using directives out of the namespace block?
```
using System.Management.Automation;
using Microsoft.DotNet.Interactive.Events;
```